### PR TITLE
Add version attribute as per Python convention

### PR DESCRIPTION
**Pull request by rasbt**
Just a simple PR so that bitsandbytes has a version attribute, which is common in Python scientific computing libraries (PyTorch, NumPy, scikit-learn, pandas, ...)

So, this PR allows the following now:

```python
>>> import bitsandbytes as bnb
>>> print(bnb.__version__)
"0.41.0"
```

### CODE REVIEW

**Comment by poedator** [comment]
@rasbt , it is not critical, but a very convenient feature. I'd like to see it merged. Could you check the reasons for the merge conflict, please?

**Comment by poedator** [comment]
@Titus-von-Koeller , would it be possible to merge this PR somehow? -- it would be helpful in situations when we test multiple bnb versions like 0.41.3.post1.

**Comment by rasbt** [comment]
> @rasbt , it is not critical, but a very convenient feature. I'd like to see it merged. Could you check the reasons for the merge conflict, please?

It was a minor issue due to to the latest BnB update overwriting a line in this PR. Should be fixed.

File: bitsandbytes/__init__.py [hunk 1]
```diff
@@ -24,4 +24,6 @@
     "optim.optimizer.MockArgs": False,
 }
 
+__version__ = "0.41.1"
```
Comment (Titus-von-Koeller) [inline bitsandbytes/__init__.py:27]: please update the version to the most recent one :)

**Review by Titus-von-Koeller — CHANGES_REQUESTED** [review]
LGTM. Please fix minor comment. Will merge in the morning.

This is a good idea. Thanks a lot for your contribution.

File: bitsandbytes/__init__.py (hunk 1 above)
Comment (rasbt) [reply bitsandbytes/__init__.py]: good call!

Comment (poedator) [reply bitsandbytes/__init__.py]: ... which changed yet again today )

Comment (rasbt) [reply bitsandbytes/__init__.py]: `0.41.3.post1` should be the current one though, correct? That's currently what we have in the PR here.

**Comment by TimDettmers** [comment]
Thank you, this is a good idea! Sorry for taking so long.

**Pull request merged**